crates/router/src/core/admin.rs

```diff
@@ -2016,7 +2016,7 @@ impl MerchantConnectorAccountCreateBridge for api::MerchantConnectorCreate {
                     connector_webhook_details.encode_to_value(
                     )
                     .change_context(errors::ApiErrorResponse::InternalServerError)
-                    .attach_printable(format!("Failed to serialize api_models::admin::MerchantConnectorWebhookDetails for Merchant: {}", business_profile.merchant_id))
+                    .attach_printable(format!("Failed to serialize api_models::admin::MerchantConnectorWebhookDetails for Merchant: {}", business_profile.merchant_id.get_string_repr()))
```

Comment:
We should not have any use of `get_string_repr` functions ( this will be removed later ), So we can use the debug impl here and in other places as well where we just want to display this information

Reply:
Fair enough, I've updated the PR to use the `Debug` impl in log messages, while keeping the other usages like cache keys or merchant-facing error messages as is. Or let me know if you would prefer me adding back the `Display` implementation for the other usages.